values/mainnet/network/space06.yaml

```diff
@@ -14,6 +14,12 @@ init:
 Lotus:
   service:
     gateway: true
+    annotations:
+      # Annotation to apply timeout from https://github.com/glifio/filecoin-iac/blob/main/k8s/kong_ingress_timeout.tf
+      konghq.com/override: kong-ingress-read-timeout
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "1234"
+      prometheus.io/path: /debug/metrics
```

Comment:
after adding this annotation block - a default annotation block will be replaced or it will be added automatically after default annotations?

Reply:
It default one will be replaced by this one

Reply:
could you look for this comment?
https://github.com/glifio/filecoin-chart/pull/68#discussion_r953934011